src/app/events/[slug]/components/EventsSection.tsx

```diff
@@ -0,0 +1,21 @@
+import type { Event } from '@/types/eventType'
+
+import { PageSection } from '@/components/PageSection'
+
+type EventsSectionProps = Pick<
+  NonNullable<Event['lumaEventsSection']>,
+  'title' | 'embedLink'
+>
```

Comment:
Looking at the zod schema, could this just be `NonNullable<Event['lumaEventsSection']>`?